templates/titian-mosaic.template

```diff
@@ -42,57 +42,57 @@ Metadata:
       - pDBMultiAZ
     ParameterLabels:
       pAppServerInstanceType:
-        default: App server instance type
+        default: App Server Instance Type
       pAppSubnetA:
-        default: Application subnet A
+        default: Application Subnet A
       pAppSubnetB:
-        default: Application subnet B
+        default: Application Subnet B
       pDBAllocatedStorage:
-        default: Database storage capacity
+        default: Database Storage Capacity
       pDBEngine:
-        default: Database engine
+        default: Database Engine
       pDBEngineLicenseModel:
-        default: Database license model
+        default: Database License Model
       pDBInstanceType:
-        default: Database instance type
+        default: Database Instance Type
       pDBMultiAZ:
         default: Multi-AZ Amazon RDS
       pDBTimezone:
-        default: Database time zone
+        default: Database Time Zone
       pDBName:
-        default: Database name
+        default: Database Name
       pDBPassword:
-        default: Database password
+        default: Database Password
       pDBSubnetA:
-        default: Database subnet A
+        default: Database Subnet A
       pDBSubnetB:
-        default: Database subnet B
+        default: Database Subnet B
       pDBUsername:
-        default: Database user/owner
+        default: Database User/Owner
       pDmzSubnetA:
-        default: Public subnet A
+        default: DMZ Subnet A
       pDmzSubnetB:
-        default: Public subnet B
+        default: DMZ Subnet B
       pDnsHostedZoneApexDomain:
-        default: Internal DNS apex domain
+        default: Internal DNS Apex Domain
       pDnsHostedZoneID:
         default: Internal DNS hosted zone ID
       pDNSName: 
-        default: Mosaic server DNS name
+        default: Mosaic Server DNS Name
       pEC2KeyPair:
-        default: Key pair for Mosaic server
+        default: Key Pair for Mosaic Server
       pLoadBalancerType:
-        default: Load balancer type
+        default: Load Balancer Type
       pSecurityGroupForWebAccess:
-        default: Web access security group
+        default: Web Access Security Group
       pVpcId:
-        default: VPC for deployment
+        default: VPC for Deployment
       pWebAccessCIDR:
-        default: Web access CIDR 
+        default: Web Access CIDR (No SG)
        
 Parameters:
   pDBTimezone:
-    Description: The time zone to set the Titian Mosaic FreezerManagement database to.
+    Description: What time zone should we set the database to? 
```

Comment:
It shouldn't be a question. It's a description. Please revert it back to earlier description.

Reply:
Corrected as requested.